R/confirmatoryfactoranalysis.R

```diff
@@ -271,7 +275,11 @@ confirmatoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
     spec$se <- "standard"
     spec$bootstrap <- TRUE
   } else {
-    spec$se <- options$seType
+    if (options$seType == "robust") {
+      spec$se <- "robust.sem"
```

Comment:
Now if I use only continuous variables, select `GLS` estimator and `robust` se, I get this

![Screenshot 2022-12-01 at 11 47 16](https://user-images.githubusercontent.com/7093546/205033489-27f19e9e-5e24-4372-8fa9-4ea4c5775740.png)

Reply:
very good catch. I hadn't checked that one. I will write an exception.

Reply:
to be fair, though, the error was already there with the old module

Reply:
I wonder if we should actually catch that error, or leave as is, for instance when we have ordinal variables, we also get an error that FIML does not work. So the user has to change that. In this case the user would have to change the SE. In my view the error message is informative enough. What do you think?